code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -364,6 +364,27 @@
 		M.bodytemperature = max(310, M.bodytemperature - (8 * TEMPERATURE_DAMAGE_COEFFICIENT)) //310 is the normal bodytemp. 310.055
 	..()
 
+/datum/reagent/consumable/pwr_game
+	name = "Pwr Game"
+	description = "The only drink with the PWR that true gamers crave."
+	id = "pwr_game"
+	color = "#9385bf" // rgb: 58, 52, 75
```

Comment:
Gonna need a flavour description for your two new drinks if and when #24323 get merged.

For reference:

Space Cola - > "cola"
Nuka Cola -> "the future"
Space Mountain Wind -> "sweet citrus soda"
Dr. Gibb -> "cherry soda"
Space-Up -> "cherry soda"
Lemon Lime -> "tangy lime and lemon soda"

To be honest, feel free to come up with anything that's appropriate and/or funny.